setup.py

```diff
@@ -29,6 +30,19 @@ def read_requirements(path):
     return [line.strip() for line in read(path).split("\n") if not line.startswith(('"', "#", "-"))]
 
 
+def get_version():
```

Comment:
I thought we specify the version in setup.py, would it not be easier to set it there and make available as a module wide variable or something? Idk how this is usually done though, or what restraints you are working under, but just curious

Reply:
The problem is you can't import setup.py from the package itself, it breaks... So neither can directly import the other.

The third option of having a standalone file say `VERSION.txt` which would be easier to parse, is that non-python files can be tricky to copy over into the installed package (I've had that problem before and it is very annoying), this felt like the least worse option which would be easily debugged if it ever broke.